library/src/main/java/com/sothree/slidinguppanel/SlidingUpPanelLayout.java

```diff
@@ -1033,10 +1035,14 @@ private boolean isViewUnder(View view, int x, int y) {
     private int computePanelTopPosition(float slideOffset) {
         int slidingViewHeight = mSlideableView != null ? mSlideableView.getMeasuredHeight() : 0;
         int slidePixelOffset = (int) (slideOffset * mSlideRange);
+
```

Comment:
The logic here looks exactly the same, can you elaborate how this is solving #694?